package/generate-package.js

```diff
@@ -0,0 +1,211 @@
+import * as ngPackage from 'ng-packagr'
+import baseTsConfig from '../tsconfig.base.json' assert { type: 'json' }
+import fs from 'fs/promises'
+import {
+  existsSync,
+  lstatSync,
+  createReadStream,
+  createWriteStream,
+  mkdirSync,
+} from 'fs'
+import path from 'path'
+import { fileURLToPath } from 'url'
+import { Transform } from 'stream'
+
+const PATH_ALIASES = baseTsConfig.compilerOptions.paths
+
+const CURRENT_DIR_PATH = path.dirname(fileURLToPath(import.meta.url))
+const PROJECT_ROOT_PATH = path.join(CURRENT_DIR_PATH, '..')
+const PACKAGE_DIST_PATH = path.join(CURRENT_DIR_PATH, './dist')
+const LIBS_SOURCE_PATH = path.join(PROJECT_ROOT_PATH, './libs')
+const LIBS_DEST_PATH = path.join(CURRENT_DIR_PATH, './libs')
+const TRANSLATIONS_SOURCE_PATH = path.join(PROJECT_ROOT_PATH, './translations')
+const TRANSLATIONS_DEST_PATH = path.join(CURRENT_DIR_PATH, './translations')
+
+/**
+ * @param {number} depth Depth value from the `libs` folder (0 means in the libs folder)
```

Comment:
One `depth` too much :-)

Reply:
it's normal actually, the first `depth` is the parameter name and afterwards is the parameter description. JSDoc weirdness